pkg/batchjob/stat.go

```diff
@@ -0,0 +1,226 @@
+package batchjob
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/robfig/cron"
+
+	"github.com/dyweb/sundial/pkg/models"
+	"github.com/dyweb/sundial/pkg/store/rdb"
+	"github.com/dyweb/sundial/pkg/store/tsdb"
+)
+
+//AddCronJob returns a function that could be passed to the cron library.
+func AddCronJob(rdbStore rdb.Store, tsStore tsdb.Store, statRange models.StatRange) func(*cron.Cron) {
```

Comment:
I am not sure why we need to return a func that accepts a cron, based on the usage, the func is not passed around and used right away with same crontabe instance, so put cron as first pointer also work and make the function easier to read

````go
func AddCronJob2(c *cron.Cron, rdbStore rdb.Store, tsStore tsdb.Store, statRange models.StatRange) {
	job := func() {
		now := time.Now()
		err := StatWorker(rdbStore, tsStore, now, statRange)
		if err != nil {
			fmt.Printf("Error during stat cron at time %s: %s\n", now, err)
		}
	}
	c.Schedule(
		cron.Every(ToDuration(statRange)),
		cron.FuncJob(job))
}
````